src/lib.rs

```diff
@@ -143,33 +191,21 @@ struct DebugInfo<'object, Endian>
 {
     debug_line: gimli::DebugLine<'object, Endian>,
     units: Vec<Unit<'object, Endian>>,
+    with_demangling: bool,
```

Comment:
Or I guess technically `DebugInfo::locate`, so that we can take `DW_AT_language` into account as @philipc suggested.